df.history.xml

```diff
@@ -791,6 +791,28 @@
         <enum-item name='SQUAD_VS_SQUAD'/>
         <enum-item name='SITE_SURRENDERED'/>
         <enum-item name='ENTITY_EXPELS_HF'/>
+
+        <enum-item name='HF_TRADED'/>
+        <enum-item name='ENTITY_HOLY_CITY'/>
+        <enum-item name='HF_GAMBLED'/>
+        <enum-item name='HF_GAINED_TITLE'/>
+        <enum-item name='ENTITY_DISSOLVED'/>
+        <enum-item name='ENTITY_PURCHASED_EQUIPMENT'/>
+        <enum-item name='HOUSE_EXTENDED'/>
+        <enum-item name='HOUSE_ACQUIRED'/>
+        <enum-item name='HF_PREACHED'/>
+        <enum-item name='RELIGIOUS_PERSECUTION'/>
+        <enum-item name='UNDERWORLD_BREACHED'/>
+        <enum-item name='ENTITY_ALLIANCE'/>
+        <enum-item name='HF_RANSOMED'/>
+        <enum-item name='HF_SOLD'/>
+        <enum-item name='UNKNOWN_127'/>
```

Comment:
Is this one that you weren't able to find in-game, or one where the class doesn't appear to exist at all? (symbols.xml should have a list of all historical event classes, since they have virtual methods)

Reply:
Got it.  Thanks, I did not know about `symbols.xml`.  I will take a re-naming pass based on those values.

Reply:
I had not found that one event in-game, but since it was only one left, I was able to find it in `symbols.xml`